### ISSUE TITLE
Introduce logging of table transformers results

### ISSUE THREAD

**Issue by ikalinin1** [vividus-framework/vividus#3291]
In order to provide user clear view of the table transformers resulting table we need to introduce logging mechanism

**Comment by GrimA9e** [comment]
In addition to logging I would suggest to unify transformers/tables/files definition order. Currently it is different, viz.:
```
Examples:
{transformer=FROM_DB, dbKey=testdb, sqlQuery=$\{sql-all-category-product-last-flag\}}
{transformer=FILTERING, byMaxRows=$\{filter-products\}}
```
and
```
Examples:
{transformer=FILTERING, column.category=.*lipstick.*}
/tables/vime/photo_p${index}.table
```
Initial source is on the opposite sides of the list

**Comment by valfirst** [comment]
Implemented in #3388
